### DISCUSSION TITLE
如何设置docker镜像启动后，节点自动连接

### DISCUSSION THREAD

**Discussion by DavidTsou** [v2rayA/v2rayA#1488] [Q&A]
现在每次机器重启都需要进到页面里，手动点一下启动才能代理

![image](https://github.com/user-attachments/assets/5912a657-4bb4-4c1e-a8d0-5c39947f52ce)

**Comment by Sunsetra** [comment]
找到一个笨办法，只需要设置重启后的计划任务，对http://127.0.0.1:2017/api/v2ray这个地址发送一个空POST请求就行了。唯一需要带的请求头是authorization，可以自己在devtool里找。
不知道有没有什么更好的办法

**Comment by ecardm** [comment]
晕，就这一个方法么。强烈建议在cli里添加这个命令。
